text/src/swam/witx/parser/package.scala

```diff
@@ -0,0 +1,17 @@
+package swam
+package witx
+package parser
```

Comment:
remove this line, the package `parser` is defined below with the package object. Here what you create is the package `swam.witx.parser.parser`.